src/pages/komodo-defi-framework/api/v20/my_recent_swaps/index.mdx

```diff
@@ -0,0 +1,295 @@
+export const title = "Komodo DeFi Framework Method: My Recent Swaps V2";
+export const description = "The my_recent_swaps method returns the data of the most recent atomic swaps executed by the Komodo DeFi Framework API node.";
+
+# my\_recent\_swaps
+
+**my\_recent\_swaps (from\_uuid page\_number=1 limit=10 my\_coin other\_coin from\_timestamp to\_timestamp)**
+
+The `my_recent_swaps` method returns the data of the most recent atomic swaps executed by the Komodo DeFi Framework API node. Please note that all filters (my\_coin, from\_timestamp, etc.) are combined using logical AND.
+
+## Arguments
+
+| Structure | Type   | Description                                                                                                                         |
+| --------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------- |
+| filter    | Object | [FilterCriteria](/komodo-defi-framework/api/common_structures/#filter-criteria) containing all the filtering criteria for the swaps |
```

Comment:
`paging_options` is not included in the arguments. It should be flattened as discussed same for `filter`, it should be flattened.  See next comment.

Reply:
fixed